src/client/hard/index.js

```diff
@@ -4,27 +4,30 @@ const msg = require('../../msg').msg;
 const DEFAULT_GAS = 300000;
 
 const api = {
-    getParams: '/harvest/parameters',
-    getModAccounts: 'harvest/accounts',
-    getDeposits: 'harvest/deposits',
-    getClaims: 'harvest/claims'
+    getParams: '/hard/parameters',
+    getModAccounts: 'hard/accounts',
+    getDeposits: 'hard/deposits',
+    getTotalDeposited: 'hard/total-deposited',
+    getBorrows: 'hard/borrows',
+    getTotalBorrowed: 'hard/total-borrowed',
+    getClaims: 'hard/claims' // FIXME
```

Comment:
I'm not sure what to do with this. I can still hit this locally against the most recent master of kava (using kvtool bootsrap). Maybe I need to rebuild my container? `http://localhost:1317/hard/claims` returns

`
{
    height: "67489",
    result: [
        {
            owner: "kava1ypjp0m04pyp73hwgtc0dgkx0e9rrydecm054da",
            deposit_denom: "ukava",
            amount: {
              denom: "hard",
              amount: "13808484802238"
            },
            claim_type: "stake"
        }
    ]
}
`

I suspect maybe this and the function below maybe just need to be deleted?

Reply:
The `getClaims` query has been moved to the Incentive module and is now called `getRewards`